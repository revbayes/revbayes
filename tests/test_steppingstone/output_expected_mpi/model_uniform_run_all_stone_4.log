Iteration	Posterior	Likelihood	Prior	alpha	bl[1]	bl[2]	bl[3]	bl[4]	bl[5]	bl[6]	bl[7]	bl[8]	bl[9]	bl[10]	bl[11]	bl[12]	bl[13]	bl[14]	bl[15]	bl[16]	bl[17]	bl[18]	bl[19]	bl[20]	bl[21]	bl[22]	bl[23]	bl[24]	bl[25]	bl[26]	bl[27]	bl[28]	bl[29]	bl[30]	bl[31]	bl[32]	bl[33]	bl[34]	bl[35]	bl[36]	bl[37]	bl[38]	bl[39]	bl[40]	bl[41]	bl[42]	bl[43]	er[1]	er[2]	er[3]	er[4]	er[5]	er[6]	gamma_rates[1]	gamma_rates[2]	gamma_rates[3]	gamma_rates[4]	pi[1]	pi[2]	pi[3]	pi[4]	pinvar	TL
0	-21552.05	-21524.34	-27.71139	0.8208895	0.140949	0.211367	0.111546	0.167103	0.284841	0.234704	0.513902	0.149389	0.116015	0.183797	0.242619	0.200932	0.178447	0.150109	0.11582	0.234204	0.249156	0.184034	0.187826	0.298169	0.174893	0.321716	0.33758	0.249763	0.0109156	0.00236343	0.191711	0.130696	0.0251865	0.204054	0.280391	0.197299	0.1314	0.00139238	0.182848	0.105829	0.0717246	0.0733294	0.0152136	0.0220801	0.12571	0.195949	0.205772	0.0264956	0.336264	0.0441417	0.0552056	0.48001	0.0578828	0.0999954	0.41541	0.962579	2.52202	0.383007	0.310662	0.0888129	0.217519	0.4243549	7.412747
10	-21482.25	-21467.56	-14.68558	0.7826357	0.13731	0.136106	0.174325	0.126906	0.0748261	0.338905	0.343797	0.0946345	0.227188	0.0324147	0.206982	0.200364	0.0967337	0.132656	0.208925	0.0720047	0.185958	0.190476	0.234478	0.202569	0.321123	0.308466	0.323737	0.100994	0.0603774	0.000951016	0.0773501	0.0722579	0.0147736	0.123825	0.192811	0.0904861	0.0233432	0.00569336	0.0637653	0.146433	0.0286633	0.182525	0.0150561	0.0321594	0.0878469	0.115147	0.296949	0.0396324	0.418157	0.0342223	0.0359366	0.391539	0.0805121	0.0918588	0.400048	0.952025	2.55607	0.340166	0.36654	0.0594549	0.233839	0.3496785	6.102292
