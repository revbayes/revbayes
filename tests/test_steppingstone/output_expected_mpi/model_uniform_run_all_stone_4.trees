Iteration	Posterior	Likelihood	Prior	psi
0	-21552.05	-21524.34	-27.71139	(((Perodicticus_potto[&index=18]:0.184034,((Nycticebus_coucang[&index=15]:0.115820,Loris_tardigradus[&index=12]:0.200932)[&index=37]:0.071725,(Otolemur_crassicaudatus[&index=16]:0.234204,Galago_senegalensis[&index=8]:0.149389)[&index=24]:0.249763)[&index=29]:0.025186)[&index=41]:0.125710,((((Microcebus_murinus[&index=14]:0.150109,Cheirogaleus_major[&index=4]:0.167103)[&index=35]:0.182848,Lepilemur_hubbardorum[&index=11]:0.242619)[&index=40]:0.022080,Propithecus_coquereli[&index=19]:0.187826)[&index=39]:0.015214,(Lemur_catta[&index=10]:0.183797,Varecia_variegata_variegata[&index=22]:0.321716)[&index=38]:0.073329)[&index=27]:0.191711)[&index=28]:0.130696,Galeopterus_variegatus[&index=23]:0.337580,((Tarsius_syrichta[&index=21]:0.174893,(((Hylobates_lar[&index=9]:0.116015,Pan_paniscus[&index=17]:0.249156)[&index=32]:0.197299,((Macaca_mulatta[&index=13]:0.178447,Chlorocebus_aethiops[&index=5]:0.284841)[&index=25]:0.010916,Colobus_guereza[&index=6]:0.234704)[&index=42]:0.195949)[&index=30]:0.204054,((Cebus_albifrons[&index=3]:0.111546,(Saimiri_sciureus[&index=20]:0.298169,Aotus_trivirgatus[&index=1]:0.140949)[&index=33]:0.131400)[&index=26]:0.002363,Callicebus_donacophilus[&index=2]:0.211367)[&index=43]:0.205772)[&index=31]:0.280391)[&index=36]:0.105829,Daubentonia_madagascariensis[&index=7]:0.513902)[&index=34]:0.001392)[&index=44];
10	-21482.25	-21467.56	-14.68558	(((((Galago_senegalensis[&index=8]:0.094635,Otolemur_crassicaudatus[&index=16]:0.072005)[&index=24]:0.100994,Loris_tardigradus[&index=12]:0.200364)[&index=37]:0.028663,Perodicticus_potto[&index=18]:0.190476)[&index=29]:0.014774,Nycticebus_coucang[&index=15]:0.208925)[&index=41]:0.087847,(Daubentonia_madagascariensis[&index=7]:0.343797,((((Propithecus_coquereli[&index=19]:0.234478,Lepilemur_hubbardorum[&index=11]:0.206982)[&index=40]:0.032159,(Cheirogaleus_major[&index=4]:0.126906,Microcebus_murinus[&index=14]:0.132656)[&index=35]:0.063765)[&index=39]:0.015056,(Varecia_variegata_variegata[&index=22]:0.308466,Lemur_catta[&index=10]:0.032415)[&index=38]:0.182525)[&index=27]:0.077350,(((Cebus_albifrons[&index=3]:0.174325,(Aotus_trivirgatus[&index=1]:0.137310,(Callicebus_donacophilus[&index=2]:0.136106,Saimiri_sciureus[&index=20]:0.202569)[&index=33]:0.023343)[&index=26]:0.000951)[&index=43]:0.296949,(((Chlorocebus_aethiops[&index=5]:0.074826,Colobus_guereza[&index=6]:0.338905)[&index=25]:0.060377,Macaca_mulatta[&index=13]:0.096734)[&index=42]:0.115147,(Hylobates_lar[&index=9]:0.227188,Pan_paniscus[&index=17]:0.185958)[&index=32]:0.090486)[&index=30]:0.123825)[&index=31]:0.192811,Tarsius_syrichta[&index=21]:0.321123)[&index=36]:0.146433)[&index=34]:0.005693)[&index=28]:0.072258,Galeopterus_variegatus[&index=23]:0.323737)[&index=44];
