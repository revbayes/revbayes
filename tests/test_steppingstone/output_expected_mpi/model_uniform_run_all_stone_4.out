state	power	likelihood
4	0.07776	-21507.5
5	0.07776	-21581.9
6	0.07776	-21580.1
7	0.07776	-21592.4
8	0.07776	-21505.2
9	0.07776	-21494.5
10	0.07776	-21467.6
